src/encoded/static/components/filegallery.js

```diff
@@ -1201,25 +1226,35 @@ DatasetFiles.defaultProps = {
 // This component only triggers the data retrieval, which is done with a search for files associated
 // with the given experiment (in this.props.context). An odd thing is we specify query-string parameters
 // to the experiment URL, but they apply to the file search -- not the experiment itself.
-export const FileGallery = ({ context, encodevers, hideGraph, altFilterDefault, showReplicateNumber }, reactContext) => (
+export const FileGallery = ({ context, hideGraph, hideControls, collapseNone, altFilterDefault, showReplicateNumber }, reactContext) => (
     <FetchedData>
         <Param name="data" url={`/search/?limit=all&type=File&dataset=${context['@id']}`} />
         <Param name="schemas" url="/profiles/" />
-        <FileGalleryRenderer context={context} session={reactContext.session} encodevers={encodevers} hideGraph={hideGraph} altFilterDefault={altFilterDefault} showReplicateNumber={showReplicateNumber} />
+        <FileGalleryRenderer
+            context={context}
+            session={reactContext.session}
+            hideGraph={hideGraph}
+            hideControls={hideControls}
+            collapseNone={collapseNone}
+            altFilterDefault={altFilterDefault}
+            showReplicateNumber={showReplicateNumber}
+        />
     </FetchedData>
 );
 
 FileGallery.propTypes = {
     context: PropTypes.object.isRequired, // Dataset object whose files we're rendering
-    encodevers: PropTypes.string, // ENCODE version number
-    hideGraph: PropTypes.bool, // T to hide graph display
+    hideGraph: PropTypes.bool, // T to only render file-table pane
```

Comment:
I think it's a little funny "true" is sometimes written out ("True") here and sometimes not ("T") but this is a minor detail.

Reply:
Yeah I used to use “T” to save space, then felt like that was needlessly obscure so I switched.